docs/src/main/sphinx/release/release-443.md

```diff
@@ -0,0 +1,46 @@
+# Release 443 (21 Mar 2024)
+
+## General
+
+* Ensure consistent formatting when casting from double or real to varchar. ({issue}`21136`)
+* Prevent query failure when partial aggregation over long decimals is pushed
+  below joins. ({issue}`21099`)
+
+## Delta Lake connector
+
+* Support automatic type coercion for timestamp type columns during in table
```

Comment:
This is not correct. It's not about automatic type coercion. It's about allowing the creating of tables when the type is specified as something other than timestamp(3). Trino will substitute for a timestamp(3) in that case. We did this for Iceberg a while ago, so we should dig up the release note from that.

Reply:
I looked back all the way to 415 .. could not find anything related

Reply:
But I reworded .. maybe the new one is better